src/test.jest.testRunner.ts

```diff
@@ -125,10 +125,6 @@ async function askRunner(
 
   const askCode = parseAskScript(source);
   const askCodeSource = askCodeToSource(askCode);
-  const askCodeSource2 = askCodeToSource(
-    fromAskScriptAst(askScriptAst, createElement)
-  );
-  // console.log({ askCodeSource1: askCodeSource, askCodeSource2 });
```

Comment:
out of the scope of this PR - please revert

Reply:
[Here](https://github.com/xFAANG/askql/pull/114#discussion_r439779267) Yonatan asked me to remove them and you agreed.
I am waiting for your decision.

Reply:
@czerwinskilukasz1 making this change would go against the title of your PR which doesn't say anything about updating the test runner. For someone who maintains the test runner it will be difficult to understand what this change was trying to achieve just by looking at the commit message. Whether or not want this change should be decided on another issue or PR.

Reply:
Fine, I will move to a separate PR.